### DISCUSSION TITLE
Requests not triggered on fast e2e test

### DISCUSSION THREAD

**Discussion by baptisteArno** [vercel/swr#1813] [Q&A] [answered]
I'm confused, I've set up an end to end test. My app is using swr hooks and it seems that at some point, when I navigate to the home page, the initial requests are not triggered if I don't wait at least for 1.5 seconds on the page before.

Let me illustrate.

On this video, the test is running at full speed and you can see that, at the end, when we're going back to the home page, initial requests (`/typebots` and `/folders`) are not triggered:
https://user-images.githubusercontent.com/16015833/151196038-4e0098a8-9610-4185-b147-981146b93eba.mp4

Here is the expected behavior:
https://user-images.githubusercontent.com/16015833/151196127-1514e8e6-7762-4e78-b36a-8a767df65a4f.mp4

`/typebots` and `/folders` requests are triggered. I've set a `waitForTimeout` of 2 seconds before going back to the home page (1 second doesn't work(?!)).

Why is it happening and why doesn't it trigger the hooks. I'm thinking maybe it pulls out data from cache, but why?

**Comment by shuding** [comment, marked as answer]
I think this is related to the default `dedupingInterval` being 2s, maybe you can change it to 0 for testing.
